azkaban-common/src/main/java/azkaban/AzkabanCommonModule.java

```diff
@@ -158,4 +163,10 @@ public FileSystem createHadoopFileSystem(final Configuration hadoopConf) {
   public QueryRunner createQueryRunner(final AzkabanDataSource dataSource) {
     return new QueryRunner(dataSource);
   }
+
+  @Provides
```

Comment:
This was a recommendation @kunkun-tang made because he said the usage was similar to QueryRunner